src/wiki/keyboard-shortcuts.md

```diff
@@ -0,0 +1,208 @@
+# Windows Keyboard shortcuts
+
+Mastering keyboard shortcuts can significantly enhance your productivity on Windows.
+
+This article will introduce you to the essential shortcuts that will help you navigate the new operating system with ease.
```

Comment:
Since its both Win10 and Win11
```suggestion
This article will introduce you to the essential shortcuts that will help you navigate the operating system with ease.
```